### DISCUSSION TITLE
Written Cantonese to Spoken Cantonese

### DISCUSSION THREAD

**Discussion by hockyy** [laubonghaudoi/jyutping.org#57] [Ideas]
Wondering whether there is a consistent tool to convert words/phrases from Written Cantonese or Standard Written Chinese to spoken variant

找到 -> 揾到
是 -> 係
的 -> 嘅

等等

Feeling really hard to study Spoken Cantonese because video-subtitle learning method is ineffective...
I need to take months and months of courses and chat with hundreds of people from carousell in order to learn spoken cantonese's and its way of typing

**Comment by hockyy** [comment]
https://github.com/BYVoid/OpenCC

**Reply by hockyy** [reply]
probably can be a reference

**Comment by laubonghaudoi** [comment]
你要嘅嘢 Google Translate 就可以解決，揀繁體中文轉粵文就得

**Reply by graphemecluster** [reply]
我估佢嘅意思係想 deterministic？或者未滿意 Google 翻譯嘅準繩度
